CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ Commit messages are in imperative mood and merges are to be avoided.
 When in doubt, or need a regresher, checking through the output of `git log` is
 highly recommended.
 
-## Commit tags
+## Using commit tags
```

Comment:
the more correct name is "commit trailers". I've seen people using "tag" in 3 contexts: 1) a git tag; 2) the prefix in the commit title; 3) in the trailer.  IMO only the first is correct. I think it's good to be unambiguous and use the correct term.